elastic/beats
# [system/process]: mark module as healthy if metrics are partially filled for single process as well.

- Bug

This PR follows https://github.com/elastic/beats/pull/40565/. It does the same thing when a single `pid` is used in config.

Closes https://github.com/elastic/elastic-agent/issues/5555

Testing steps:
- Build `agentbeat` from this branch.
- Place the `agentbeat` binary under `elastic-agent/components`
- Run the elastic agent in unprivileged mode and enabled the `system.process` dataset for any privileged process.
- You should see status as healthy.
